docs/ruleset_functions/get_loop_zone_list_w_area.md

```diff
@@ -0,0 +1,51 @@
+
+## get_loop_zone_list_w_area
+
+Description: Get the list of zones and their total floor area served by each CHW or HHW loop in a RMR.
+
+Inputs:  
+- **RMR**: The RMR that needs to get the list of zones with their total floor area served by each CHW or HHW loop.
+
+Returns: 
+- **loop_zone_list_w_area_dictionary**: A dictionary that saves the list of zones and the total floor area served by each CHW or HHW loop, i.e. {loop_1.id: {"ZONE_LIST": [zone_1.id, zone_2.id, zone_3.id], "TOTAL_AREA": 10000}, loop_2.id: {"ZONE_LIST": [zone_10.id], "TOTAL_AREA": 500}}
+
+Logic:  
+
+- For each zone in RMR: `for zone in RMR...zones:`
+
+  - Check if zone is connected to any terminal: `if zone.terminals:`
+
+    - Get zone total floor area: `zone_area = SUM(space.floor_area for space in zone.spaces)`
+
+    - For each terminal serving zone: `for terminal in zone.terminals:`
+
+      - Check if terminal is connected to heating hot water loop: `if terminal.reheat_from_loop:`
+
```

Comment:
reheat_from_loop seems to be renamed to heating_from_loop

Reply:
We can alternatively calculate area of all zones served by HVAC system that had heatingVentilationAirConditioningSystem.preheat_system.hot_water_loop. THen the check would be analogous to what is done for cooling.

Reply:
Updated.